docs/source/reference/numpysupported.rst

```diff
@@ -582,6 +582,49 @@ Modules
 ``random``
 ----------
 
+Generator Objects
+'''''''''''''''''
+Numba supports `numpy.random.Generator()` objects and their respective
```

Comment:
IIRC the correct markup in our Sphinx setup for things that should be in fixed-font is to use two backticks ` `` ` instead of a single one on either side of the term. Looking at how this renders, the technical terms are in italics:

<img width="1114" alt="Screen Shot 2022-05-17 at 12 02 09" src="https://user-images.githubusercontent.com/41563/168786345-74eed098-d629-4e30-9bc9-ee45e758fe82.png">